src/parser/core/CASTS_THAT_ARENT_CASTS.ts

```diff
@@ -72,6 +73,11 @@ const spells: number[] = [
   SPELLS.CARVED_IVORY_KEEPSAKE.id,
   SPELLS.ANIMA_FIELD_EMITTER_BUFF.id,
   //endregion
+
+  // CLASSIC
+  // region Mage
+  CLASSIC_SPELLS.MISSILE_BARRAGE.id,
```

Comment:
I think we'll need a classic variation of this, so you don't accidentally set a retail spellid to not be a cast.

Reply:
IDs don't clash between retail and classic. There are many files we use retail and classic IDs in.

Reply:
There are plenty of spell id overlaps in general between classic and retail where a given spell works fundamentally different, a split of CASTS_THAT_ARENT_CASTS would make such a possible clash, not possible.